lib/server-code/runners/cloud-workers-broker.js

```diff
@@ -15,6 +16,31 @@ const Events = {
   TASK_PROCESSED     : 'TASK_PROCESSED',
 }
 
+function getKilledWorkerDetails(worker) {
+  const { appId, task } = worker
+  const details = [`appId: ${appId}`]
+
+  if (task.deploymentModelName) {
+    details.push(`deployed model: ${task.deploymentModelName}`)
+  }
+
+  if (task.className) {
+    const taskClassName = task.className.match(/\.\w+/)[0].substring(1)
+    details.push(`service name: ${taskClassName}`)
+  }
+
+  if (task.method) {
+    details.push(`method: ${task.method}`)
+  }
+
+  if (task.eventId) {
+    const event = ServerCodeEvents.get(task.eventId)
+    details.push(`handler-event-name: ${event.provider.id}(${event.name})`)
+  }
+
+  return details.join(', ')
```

Comment:
let's also add information about a file, see property `task.provider`

Reply:
19:26:33.366 [41365] Worker killed. Worker expired due to heartbeat timeout (10000ms), the last ping was: 2020-10-08T16:26:19.083Z [appId: 1122C599-6A3F-4AEB-8CB3-26EB8CDDF2BA, model: default, handler: timer(WWWWWWWWWWWWWWW), file-path: timers/wwwwwwwwwwwwwww.js]